Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -427,6 +427,9 @@ public class LottieAnimationLayer: CALayer {
   /// Will inform the receiver the type of rendering engine that is used for the layer.
   public var animationLayerDidLoad:((_ animationLayer: LottieAnimationLayer, _ renderingEngine: RenderingEngineOption) -> Void)?
 
+  /// The parent `LottieAnimationView` of this layer, if applicable.
+  public internal(set) weak var animationView: LottieAnimationView?
+
```

Comment:
Why do we need this, asking to learn?

Reply:
There was an existing public API for this:

```swift
public var animationView: LottieAnimationView? {
    set { rootAnimationLayer?.animationView = newValue }
    get { rootAnimationLayer?.animationView }
}
```

I was thinking we can't remove an existing public API due to backwards compatibility, so I just changed it to a stored property.

Reply:
Technically I guess we could remove this, because there hasn't been an official release that includes `LottieAnimationLayer`, so we can still make breaking changes here. 

@eromanc, are you using this for anything? I imagine not since Oragami just uses CALayers.

If there are no consumers of this we could just remove it completely.

Reply:
let me confirm with you tomorrow @calda but I assume we are not using that.

Reply:
Confirmed. We are not using that property in Origami.

Reply:
Great! In that case I'll just remove this API.